src/hash_algorithms.jl

```diff
@@ -11,21 +11,9 @@
 
 Returns the updated hash state given a set of bytes (either a tuple or array of UInt8
 values).
-
```

Comment:
We no longer need the method with `context` here, because that was used to call `StableHashTraits.write` which no longer exists.